### PR TITLE
Clean up JSON written to clipboard for log

### PR DESCRIPTION
**Pull request by ccschmitz**
## Summary

Copying to clipboard includes some unnecessary details of a log. Also, everything under `logAttributes` should be included as top-level keys, similar to what the user sees in the UI.

## How did you test this change?

Local click test.

## Are there any deployment considerations?

N/A

### CODE REVIEW

File: frontend/src/pages/LogsPage/LogsTable/LogDetails.tsx [hunk 1]
```diff
@@ -57,6 +58,11 @@ export const LogDetails = ({ row, queryTerms }: Props) => {
 	const expandable = Object.values(logAttributes).some(
 		(v) => typeof v === 'object',
 	)
+	const attributesShownWhenPresent = {
```
Comment (et) [inline frontend/src/pages/LogsPage/LogsTable/LogDetails.tsx:61]: @ccschmitz - per #4672, can adjust this to `ReservedLogKey`: https://github.com/highlight/highlight/blob/main/backend/private-graph/graph/schema.graphqls#L533-L541

Comment (et) [reply frontend/src/pages/LogsPage/LogsTable/LogDetails.tsx:61]: We might need to add `message` to that enum.

Comment (Vadman97) [reply frontend/src/pages/LogsPage/LogsTable/LogDetails.tsx:61]: with #4672, we'll want `Source` and `ServiceName` as part of this as well.

**Review by Vadman97 — APPROVED** [review]

**Comment by ccschmitz** [comment]
@Vadman97 @et thanks for the heads up on that. I revised this logic in https://github.com/highlight/highlight/pull/4711/commits/437502e990936f6ea806f7da23a26557708b096b and used the `ReservedLogKey` type to enforce the object I created and iterated over stays in sync with what's on the server.

File: frontend/src/pages/LogsPage/LogsTable/LogDetails.tsx [hunk 2]
```diff
@@ -57,6 +57,15 @@ export const LogDetails = ({ row, queryTerms }: Props) => {
 	const expandable = Object.values(logAttributes).some(
 		(v) => typeof v === 'object',
 	)
+	const reservedLogAttributes: {
+		[key in ReservedLogKey]: Maybe<string> | undefined
```
Comment (et) [inline frontend/src/pages/LogsPage/LogsTable/LogDetails.tsx:61]: Awesome 👏

**Review by et — APPROVED** [review]

**Pull request merged**